www/situm.js

```diff
@@ -409,6 +409,17 @@ var Situm = {
   updateNavigationWithLocation: function (location, cb, error) {
     exec(cb, error, PLUGIN_NAME, 'updateNavigationWithLocation', [location]);
   },
+    /**
+   * For internal use only
+   * @description Informs NavigationManager object the change of the user's location.
+   * @param {ExternalNavigation} externalNavigation New Location of the user. If null, nothing is done.
```

Comment:
"Nes Location of the user". ExternalNavigation é sólo localización? 
Si é un método interno, quizais mereza a pena decir algo como "Navigation payload" (ou polo estilo), en plan xenérico, pero creo que "localización" quédase corto non?